.github/workflows/linux.yml

```diff
@@ -32,7 +32,7 @@ jobs:
     defaults:
```

Comment:
please change the title of workflow above:
Tests on Linux (Ubuntu 22.04, Python 3.11) -> Tests on Linux (Ubuntu 20.04, Python 3.11)

Also, please change machines for other jobs in the flow. And use ubuntu-20.04-8-cores for TensorFlow Hub Model tests

Reply:
yes, thanks, fixed